src/hammer-vlsi/cli_driver.py

```diff
@@ -360,6 +400,11 @@ def args_to_driver(self, args: dict,
         if top_module is not None:
             config['synthesis.inputs.top_module'] = top_module
 
+        # Waveform files
+        waveforms = args['waveform']
+        if isinstance(waveforms, List) and len(waveforms) > 0:
+            config['power.inputs.waveform_files'] = waveforms
```

Comment:
I think this should follow the `verilogs` example more closely. i.e. including the `list` conversion